whylog/tests/tests_client/test_files/001_most_basic/parsers.yaml

```diff
@@ -1,6 +1,10 @@
 ---
+class: RegexParser
```

Comment:
This should definitely be a default setting, so that it doesn't have to be explicitly set for every parser.